awx/ui/src/screens/NotificationTemplate/NotificationTemplateDetail/NotificationTemplateDetail.js

```diff
@@ -364,18 +403,22 @@ function NotificationTemplateDetail({ template, defaultMessages }) {
             >
               {t`Edit`}
             </Button>
-          )}
-        {summary_fields.user_capabilities &&
-          summary_fields.user_capabilities.delete && (
-            <DeleteButton
-              name={template.name}
-              modalTitle={t`Delete Notification`}
-              onConfirm={deleteTemplate}
-              isDisabled={isLoading}
-            >
-              {t`Delete`}
-            </DeleteButton>
-          )}
+            <Button
+              onClick={sendTestNotification}
+              variant="secondary"
```

Comment:
If the test notification is loading or running we could disable this button to avoid an extra test to be triggered.